docs/EBYTE_E77_MBL.md

```diff
@@ -0,0 +1,39 @@
+# mLRS Documentation: EBYTE E77 MBL Series Boards #
+
+([back to main page](../README.md))
+
+EBYTE MBL Series Boards are an option for building mLRS equipment. These boards use the EBYTE E77 module and are available in both 868/915 MHz and 433 MHz/70 cm versions. However, these boards are not perfect since their pins are not ready-made for the purposes of mLRS. So, some tweaking and (easy) soldering is required.
```

Comment:
technically not correct, since the MBL series comprises all sorts of boards, also those which do not have an E77...
so maybe "EBYTE E77 MBL boards...." ?

EBYTE or EByte?

Reply:
- Makes sense, all references are now 'EBYTE E77 MBL Boards'
- Have seen all variations of official docs 'EBYTE', 'eByte', 'EByte', 'Ebyte'.  The logo on the website is 'EBYTE' so I went with that. :)